frontend/src/components/NavBar/PrintImage/__snapshots__/index.test.tsx.snap

```diff
@@ -200,11 +200,16 @@ exports[`renders as expected 1`] = `
         inputprops="[object Object]"
         maxrows="4"
         multiline="true"
-        value="Layers represent data on 2023-11-22.  The designations employed and the presentation of material in the map(s) do not imply the expression of any opinion on the part of WFP concerning the legal of constitutional status of any country, territory, city, or sea, or concerning the delimitation of its frontiers or boundaries."
+        value="Layers represent data on 2023-11-23.  The designations employed and the presentation of material in the map(s) do not imply the expression of any opinion on the part of WFP concerning the legal of constitutional status of any country, territory, city, or sea, or concerning the delimitation of its frontiers or boundaries."
```

Comment:
@echaidemenos let's fix the date in tests so that it doesn't change every day no?